interfaces/builtin/udisks2.go

```diff
@@ -39,6 +40,8 @@ const udisks2BaseDeclarationSlots = `
       slot-snap-type:
         - app
         - core
+      slot-attributes:
+        udev-file: $MISSING
```

Comment:
it's probably a good idea to test this slightly more complex base-declaration with dedicated cases for udev-file present/absent in interfaces/policy/basedeclaration_test.go baseDeclSuite.TestSlotInstallation test

Reply:
Done.